README.md

```diff
@@ -7,65 +7,75 @@ SweFreq - Swedish Frequency database
 Running on a production system
 ------------------------------
 
-The application has only been tested with python 3.5.2. It will most likely work with at least other 3.5 versions.
+The application is targeted at python 3.6. It will most likely work with later versions as well.
 
-`virtualenv` is not a requirement but it will help you to install the application.
+`venv` is not a requirement but it will help you to install the application.
```

Comment:
I'm using the `virtualenv` binary to create virtual environments, or has it changed name to `venv`?

Reply:
`venv` is included as a module with python 3, and thus I thought it would be easier to refer to that module instead of `virtualenv`.